_posts/2022-02-14-datafusion-7.0.0.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: Apache Arrow DataFusion 7.0.0 Release
+date: "2022-02-14 00:00:00"
```

Comment:
```suggestion
date: "2022-02-17 00:00:00"
```

I plan to release 7.0.0 in about 1 hour (after the required 72 hour voting time period has elapsed)

Reply:
Updating now